README/DEVELOPMENT.md

```diff
@@ -0,0 +1,69 @@
+# ROOT Development Practice
+
+*Vassil Vassilev*
+
+## Overview
+
+The development of ROOT almost exclusively happens using the [pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+model of github. A pull request (PR) should contain a set focused changes
+organized in one or more [atomic commits](https://en.wikipedia.org/wiki/Atomic_commit#Revision_control).
+PRs should be well-documented and well-tested in order to allow other community
+members to use, maintain and modify. If the PR contains performance critical
+code consider writing a benchmark against the [rootbench repository](https://github.com/root-project/rootbench).
+
+
+## Quality Assurance
+
+Each contribution should contains developer documentation in the form of code
+comments and sufficient amount of tests in the form of unit and/or integration
+tests. Unit tests are relatively small and quick programs focused to check if
+small pieces of code and API work as expected. Integration tests are checks
+which ensure the synergy between different (unit tested) components. Put in
+practice, unit tests verify (member) function behavior whereas integration tests
+check classes and their cooperation. The boundary between both kinds of testing
+is blur.
+
+ROOT has support for both kinds of tests in the [roottest repository](https://github.com/root-project/roottest)
+and supports "inline" unit tests in each component `test` folder. Unit testing
+uses the [GTest and GMock](https://github.com/google/googletest) infrastructure
+along with a small ROOT-specific extensions located in
```

Comment:
```suggestion
along with small ROOT-specific extensions located in
```

Reply:
@hageboeck, thanks for finding all those typos I created for such a small doc ;)